client.go

```diff
@@ -94,6 +94,11 @@ func (c *Client) Store(req Request, dest string) error {
 		return err
 	}
 	defer resp.Body.Close()
+
+  // Check for 2XX Status Codes
+	if resp.StatusCode < http.StatusOK || resp.StatusCode >= http.StatusMultipleChoices {
```

Comment:
Why not simply check `res.StatutsCode != http.StatusOK`?

Reply:
Indeed I was not aware whether `gotenberg` returns any non-200 (2XX) status codes upon successful conversion to pdf.
If the status code is expected to be 200 always, you can let me know and I can make the required change.

Reply:
Yes the expected status code is always excepted to be 200 😄

Reply:
Done.